openapi-spec/swagger.yaml

```diff
@@ -958,6 +958,57 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorSpec'
+  '/v1/storages/{storage_id}/alerts/sync':
```

Comment:
I feel this has to be GET {storage_id}/alerts .  here we are not doing any create /sync operation , just listing the laerts is it?

Reply:
No, we dont list alert here, we do sync through export and user will not get any output response other than code